src/client/index.js

```diff
@@ -1,8 +1,55 @@
 import React from 'react';
 import { render } from 'react-dom';
-import * as App from './App';
+import { Provider } from 'react-redux';
+import { createStore } from 'redux';
+import ArticlesApp from './App/Reducers';
```

Comment:
`./App/Reducers` is exporting reducers. `ArticlesApp` just doesn't seem like a correct name.